.github/ISSUE_TEMPLATE/api-suggestion.md

```diff
@@ -0,0 +1,54 @@
+---
+name: API proposal
+about: Propose a change to the public API surface
+title: ''
+labels: [Feature Request, Concept-API]
+assignees: ''
+
+---
+
+## Background and Motivation
+
+<!--
+We welcome API proposals! We have a process to evaluate the value and shape of new API. There is an overview of our process [here](https://github.com/dotnet/roslyn/blob/main/docs/contrbuting/API%20Review%20Process.md). This template will help us gather the information we need to start the review process.
+First, please describe the purpose and value of the new API here.
+-->
+
+## Proposed API
+
+<!--
+Please provide the specific public API signature diff that you are proposing. For example:
+```diff
+namespace Microsoft.CodeAnalysis.Operations
+{
+     public class ISwitchExpressionOperation
+     {
++        public bool IsExhaustive { get; }
+     }
+```
+You may find the [Framework Design Guidelines](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/framework-design-guidelines-digest.md) helpful.
+-->
```

Comment:
Having the example diff snippet inside the comment tripped me up in https://github.com/dotnet/roslyn/issues/53410 because I didn't notice, and my diff didn't show up. Then I went and deleted all of the comments, and accidentally removed the end code block marker. I don't see any problem with having the snippet be outside the comment.